webpack-article-1/first-draft.md

```diff
@@ -2,10 +2,9 @@
 
 ## Introduction
 
-Clio's main software offering received a revamp in 2017. Our legacy application is a Ruby on Rails application that uses ERB templates to serve up the views. The revamp, codenamed Apollo, is a single page application compiled together using Webpack. [Could potentially mention that it's powered mostly by our public API]
+Clio's main software offering received a revamp in 2017. Our legacy application, dubbed Themis, is a Ruby on Rails application that uses ERB templates to serve up the views. The revamp, codenamed Apollo, is a single page application compiled together using Webpack and backed by our [public API](https://app.clio.com/api/v4/documentation).
 
 Our Production Engineering team set up an automated system for building newer versions of our app. We use BuildKite as our continuous integration (CI) platform. The system is pretty slick: whenever a developer checks code into our repository, a new build is created and run automatically, compiling the source code for both Themis and Apollo.
-[You don't define what Themis is]
```

Comment:
Ooops, my bad. you mention Themis in the opening paragraph

Reply:
There are some weird merging artifacts going on, I definitely had it in there at one point, then it got deleted accidentally due to a merge. I've added it back as you have pointed it out, thanks! 😄